github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1467,6 +1467,8 @@ periodics:
             value: "true"
           - name: KUBEVIRT_E2E_SKIP
             value: Multus|SRIOV|GPU|Macvtap|MediatedDevices
+          - name: KUBEVIRT_STORAGE
+            value: rook-ceph-default
```

Comment:
Yes, I fully agree, thanks @vasiliy-ul

Reply:
If it's easy enough to simply clone the current v2 lane with this added environment variable - than yes it sounds a good idea. This way I can test it without harassing everyone.

I can also create PRs to remove this new lane and inject the environment variable to the current lane when https://github.com/kubevirt/kubevirt/pull/6042 is merged.

Reply:
Currently the presubmit job is not set as mandatory and it needs to be triggered manually on a PR.... But thinking of it a bit more, I agree that better to create a separate lane if that is okay (at least for the time being while the PR is on review and not yet merged).

Reply:
I agree it won't make a lot of difference since almost no one runs v2 lane which does not run by default, so your point is valid, but it it's very easy to clone it maybe it's better just to be sure we're not interfering with someone else.. I'm okay with both approaches :)